### DISCUSSION TITLE
Gmsh in Neko

### DISCUSSION THREAD

**Discussion by Mehdiz007** [ExtremeFLOW/neko#1649] [Q&A]
Hi all. I am new to Neko. Is there any possibility of importing msh files from other packages to Neko (Like gmsh2nek in Nek5000)?

**Comment by timofeymukha** [comment]
Yeah. If you have a Nek5000 mesh, you can convert it to a Neko one with `rea2nbin`

**Reply by timofeymukha** [reply]
(despite the name the Nek5000 one is better to be in the .re2 format)

**Reply by Mehdiz007** [reply]
Thanks for the reply. So I can convert the re2 file to Neko acceptable format using rea2nbin.

**Reply by timofeymukha** [reply]
Correct! That is pretty much the way to go for now. We hope to have a native reader for maybe .msh from gmesh, but it is not there yet.

**Reply by Mehdiz007** [reply]
Thanks. It was a great help.
